docs/self-managed/platform-deployment/helm-kubernetes/guides/air-gapped-installation.md

```diff
@@ -0,0 +1,147 @@
+    |_ operate
+    |_ tasklist
+```
+
+- Keycloak is a dependency for Camunda Identity and PostgreSQL is a dependency for Keycloak.
+- Elasticsearch is a dependency for Zeebe, Operate, Tasklist, and Optimize.
+
+The values for the dependencies Keycloak and PostgreSQL can be set in the same hierarchy:
+
+```yaml
+identity:
+  [identity values]
+  keycloak:
+    [keycloak values]
+    postgresql:
+      [postgresql values]
+```
+
+## Push images to your repository
+
+All the [required images](#required-images) need to be pushed to your repository using the following steps:
+
+1. Tag your image using the following command (replace `<IMAGE ID>`, `<DOCKER REPOSITORY>`, and `<DOCKER TAG>` with the corresponding values.)
+
+```
+docker tag <IMAGE_ID> example.jfrog.io/camunda/<DOCKER_IMAGE>:<DOCKER_TAG>
+```
+
+2. Push your image using the following command:
+
+```
+docker push example.jfrog.io/camunda/<DOCKER_IMAGE>:<DOCKER_TAG>
+```
+
+## Deploy Helm charts to your repository
+
+You must deploy the [required Helm charts](#required-helm-charts) to your repository.
+For example, in JFrog you can do this via the Artifactory's REST API.
+
+```
+curl -u <USER>:<PASSWORD> -T <PATH_TO_FILE> "https://example.jfrog.io/artifactory/camunda-platform/<TARGET_FILE_PATH>"
+```
+
+Alternatively, you can set up remote repositories (for example, with URL [https://helm.camunda.io](https://helm.camunda.io) for Camunda charts, [https://helm.elastic.co](https://helm.elastic.co) for Elastic).
+
+### Add your Helm repositories
+
+You must add your Helm chart repositories to use the charts:
+
+```
+helm repo add camunda https://example.jfrog.io/artifactory/api/helm/camunda-platform
+helm repo add elastic https://example.jfrog.io/artifactory/api/helm/elastic
+helm repo add bitnami https://example.jfrog.io/artifactory/api/helm/bitnami
+helm repo update
+```
+
+### Helm Chart values
+
+In a custom values file, it is possible to override the image repository and the image tag.
+
+```yaml
+zeebe:
+  image:
+    repository: example.jfrog.io/camunda/zeebe
+    # e.g. work with the latest versions in development
+    tag: latest
+zeebe-gateway:
+  image:
+    repository: example.jfrog.io/camunda/zeebe
+    tag: latest
+elasticsearch:
+  image: example.jfrog.io/elastic/elasticsearch
+  imageTag: 7.16.3
+identity:
+  image:
+    repository: example.jfrog.io/camunda/identity
+    ...
+  keycloak:
+    image:
+      repository: example.jfrog.io/bitnami/keycloak
+      ...
+    postgresql:
+      image:
+        repository: example.jfrog.io/bitnami/postgres
+        ...
+operate:
+  image:
+    repository: example.jfrog.io/camunda/operate
+    ...
+tasklist:
+  image:
+    repository: example.jfrog.io/camunda/tasklist
+    ...
+optimize:
+  image:
+    repository: example.jfrog.io/camunda/optimize
+    ...
+```
+
+Afterwards, you can deploy Camunda Platform using Helm and the custom values file.
+
+```
+helm install my-camunda-platform camunda-cloud/camunda-platform -f values.yaml
```

Comment:
in the [previous steps](https://github.com/camunda/camunda-platform-docs/pull/1403/files#diff-487e61948ce39b990349e3ad3ae42dd507d492c842bd72f601eb0ee8cbf15074R94) the Helm repo is called `camunda` not `camunda-cloud`.